include/TSettings.h

```diff
@@ -0,0 +1,335 @@
+// BeamMP, the BeamNG.drive multiplayer mod.
```

Comment:
Please rename into `Settings.h` and move all non-template fields and methods into a `src/Settings.cpp` file